src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -631,14 +635,22 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private class CloseAction extends AbstractAction {
-
         private CloseAction() {
             super(Localization.lang("Close window"), IconTheme.JabRefIcon.CLOSE.getSmallIcon());
             putValue(Action.SHORT_DESCRIPTION, Localization.lang("Close window"));
         }
 
         @Override
         public void actionPerformed(ActionEvent e) {
+            Map<String,String> cleanedEntries = entry
+                    .getFieldMap()
+                    .entrySet()
+                    .stream()
+                    .collect(Collectors.toMap(f -> f.getKey(), f-> BracesCorrector.apply(f.getValue())));
```

Comment:
I think you can use method references for both getKey and getValue here:
https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html

Reply:
For getKey yes. I couldn't get it to work for getValue tho and I'm not sure if you can use method references if more than one method is called.